pyspcwebgw/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Python wrapper for the Lundix SPC Web Gateway REST API."""
+import asyncio
 import logging
 from urllib.parse import urljoin
-from asyncio import ensure_future
```

Comment:
I dont think this PR was supposed to remove line 1 and add line 4...

Reply:
I think I was wrong, this looks like the code I am running since the PR was created, and it makes sense to me.